docs/src/main/sphinx/installation/kubernetes.rst

```diff
@@ -0,0 +1,342 @@
+=============================
+Trino on Kubernetes with Helm
+=============================
+
+`Kubernetes <https://kubernetes.io>`_ is a container orchestration platform that
+allows you to deploy Trino and other applications in a repeatable manner across
+different types of infrastructure. This can range from deploying on your laptop
+using tools like `kind <https://kind.sigs.k8s.io>`_ and
+`Helm <https://helm.sh>`_, to running on a managed Kubernetes service on cloud
+services like
+`Amazon Elastic Kubernetes Service <https://aws.amazon.com/eks>`_,
+`Google Kubernetes Engine <https://cloud.google.com/kubernetes-engine>`_, and
+`Azure Kubernetes Service <https://azure.microsoft.com/en-us/services/kubernetes-service>`_.
+
+The fastest way to run Trino on Kubernetes is to use the
+`Trino Helm chart <https://github.com/trinodb/charts>`_. Helm is a package
+manager for Kubernetes applications that allows for simpler installation and
+versioning by templating Kubernetes configuration files. This allows you to
+prototype on your local or on-premise cluster and use the same deployment
+mechanism to deploy to the cloud to scale up.
+
+Prerequisites
+-------------
+
+* Have a running Kubernetes cluster with a version that falls within
+  `the Kubernetes support window <https://kubernetes.io/releases/>`_.
+
+  * For local deployments, we recommend using
```

Comment:
```
28:28   warning     Try to avoid using              Google.We       
                    first-person plural like 'we'.  
```
```suggestion
  * For local deployments, you can use
```

Reply:
I agree with Jan's language anyway - we don't "recommend" anything in this case. Up to the user how they want to do it - we can just document some of the many possible ways.